### DISCUSSION TITLE
Where is the discussion for F#

### DISCUSSION THREAD

**Discussion by Xyncgas** [dotnet/csharplang#6377] [Q&A] [answered]
Is F# a Microsoft project just like Microsoft Made C#
Is there a repository for F# language just like this one

**Comment by Xyncgas** [comment]
I recently picked up F# while I am playing with MAUI I think I found myself a very fun toy

**Comment by theunrepentantgeek** [comment, marked as answer]
F# lives here:

https://github.com/dotnet/fsharp

**Reply by huoyaoyuan** [reply]
This should be the compiler repo. Discussions for new features should happen in https://github.com/fsharp/fslang-suggestions and https://github.com/fsharp/fslang-design .

**Reply by Xyncgas** [reply]
Thanks
